user_flair/new_hero_master_flair.scss

```diff
@@ -0,0 +1,188 @@
+.flair-h-master-Abathur::before{
```

Comment:
We could use the `h-m` prefix or `hm` prefix for these. Shorter class names plus still clear.